js/ui/handler/touch_zoom_rotate.js

```diff
@@ -39,6 +39,13 @@ TouchZoomRotate.prototype = {
         this._startVec = p0.sub(p1);
         this._startScale = this._map.transform.scale;
         this._startBearing = this._map.transform.bearing;
+        this._scalingSignificantly = false;
+        this._rotatingSignificantly = false;
+        this._blockRotation = undefined;
+
+        // TODO(jliebrand): Move these to config options on the map?
```

Comment:
Nah, this class is a good place for `_significantScaleThreshold` and `_significantRotateThreshold`. We should hold off on making them user configurable unless an good use case arises.